File: _ref_ios/bbplay-ios/BBPlay/Core/Modules/TabBar/Tabs/Reservation/Reservation/ReservationNetworkAPI/NetworkModels/BookingResponse.swift
import Foundation

struct BookingsResponse: Decodable {
    let code: Int
    let message: String
    let data: [BookingsDataResponse]?
}

struct BookingsDataResponse: Decodable {
    let memberOfferId: Int
    let productPcName: String
    let memberAccount: String
    let productAvailableDateLocalFrom: String
    let productAvailableDateLocalTo: String
    let productMins: Int
    let productDescription: String
    
    enum CodingKeys: String, CodingKey {
        case memberOfferId = "member_offer_id"
        case productPcName = "product_pc_name"
        case memberAccount = "member_account"
        case productAvailableDateLocalFrom = "product_available_date_local_from"
        case productAvailableDateLocalTo = "product_available_date_local_to"
        case productMins = "product_mins"
        case productDescription = "product_description"
    }
}

File: _ref_android/bbplay-android/app/src/main/java/mobi/blackbears/bbplay/screens/booking/data/model/booking/BookingResponse.kt
package mobi.blackbears.bbplay.screens.booking.data.model.booking

import com.google.gson.annotations.SerializedName

data class BookingResponse(
    @SerializedName("member_offer_id")
    val memberOfferId: Long,

    @SerializedName("product_pc_name")
    val productPcName: String,

    @SerializedName("member_account")
    val memberAccount: String,

    @SerializedName("product_available_date_local_from")
    val startSession: String,

    @SerializedName("product_available_date_local_to")
    val endSession: String,

    @SerializedName("product_mins")
    val productMinutes: Int
)